frontend/src/pages/Onboarding.tsx

```diff
@@ -0,0 +1,172 @@
+import { useState, ChangeEvent } from "react";
+import { Button, ButtonGroup, Flex, Spacer, Input, Heading, VStack, InputGroup, Textarea, FormLabel } from '@chakra-ui/react';
+
+const Onboarding = () => {
+    let [userInfo, setUserInfo] = useState({
+        role: '',
+        bio:'',
+        expertise: '',
+        learningGoals: '',
+        experience: '',
+        jobTitle: '',
+        interests: '',
+        hourlyRate: ''
+    });
+
+    const { role } = userInfo;
+
+    const handleChange = (e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement >) => {
+        setUserInfo({...userInfo, [e.target.name]: e.target.value})
+    };
+
+    const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+        //POST request logic goes here
+        e.preventDefault();
+        window.location.href='/home';
+    };
+
+    return (
+        <Flex
+            maxWidth='30%'
+            gap={5}
+            flexDirection='column'
+            margin='auto'
+        >
+            <Flex
+                border='1px'
+                borderRadius='10'
+                borderColor='lightgray'
+                p={5}
+            >
+                <VStack
+                    textAlign='center'
+                    flexWrap='wrap'
+                >
+                    <Heading
+                        fontSize='clamp(1rem, -0.875rem + 8.333vw, 2rem)'
+                    >
+                        Are you joining MentorShip as a mentor or mentee?
+                    </Heading>
+                    <Spacer />
+                    <Spacer />
+                    <ButtonGroup
+                        size='lg'
+                        flexWrap='wrap'
+                    >
+                        <Button onClick={() => setUserInfo({...userInfo, role: 'mentor'})}>Mentor</Button>
+                        <Button onClick={() => setUserInfo({...userInfo, role: 'mentee'})}>Mentee</Button>
+                    </ButtonGroup>
+                </VStack>
+            </Flex>
+     
+            {role && 
+                <Flex
+                    border='1px'
+                    borderRadius='10'
+                    borderColor='lightgray'
+                    flexDirection='column'
+                    rowGap={5}
+                    textAlign='center'
+                    p={5}
+                >
+      
+                        <Heading>{(role === 'mentee' && 'Mentee') || (role === 'mentor' && 'Mentor')}</Heading>
```

Comment:
Lets turn this into one statement using a ternary

Reply:
This has been removed as separate UIs were created for Mentor and Mentee.